### PR TITLE
fixed broken code blocks and fixed indentation (made it consistent)

### PR DESCRIPTION
**Pull request by basdp**

### CODE REVIEW

**Comment by basdp** [comment]
@CarmenPopoviciu bump :-)

**Comment by basdp** [comment]
@CarmenPopoviciu Are you still alive? :-( I'm worried!

**Comment by robinj** [comment]
Is this commit just a reformat of the code or are there changes? I am trying to understand what exactly has changed, since mass reformatting of files tends to cause the diff tool to be less helpful.

**Comment by basdp** [comment]
No, this is just formatting and styling. No changes to the code or text at all.

**Comment by LeviSchuck** [comment]
Why isn't this accepted and merged yet?

**Review by robinj — CHANGES_REQUESTED** [review]
Please split the reformatting and actual changes into two commits/PRs to make the history clearer

**Comment by basdp** [comment]
@robinj there are no actual changes.

File: README.md [hunk 1]
```diff
@@ -374,58 +374,58 @@ but since that's all encapsulated in one place, the task is much more manageable
...
+         }
+       }
+     },
+     e2e.dev: {
        options: {
          args: {
-           baseUrl: local_base_url,
-           seleniumAddress: local_Se,
+           baseUrl: dev_base_url,
+           seleniumAddress: dev_Se,
```
Comment (robinj) [inline README.md:423]: Please separate this change into a separate commit

**Comment by robinj** [comment]
@basdp I have commented where the change is

**Comment by LayZeeDK** [comment]
#40 fixed the rendering of the code blocks.

**Comment by robinj** [comment]
@layzeedk agreed

**Pull request closed without merging**